specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -3408,6 +3411,65 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/retrieveBootDiagnosticsData": {
+      "post": {
+        "tags": [
+          "VirtualMachines"
+        ],
+        "operationId": "VirtualMachines_RetrieveBootDiagnosticsData",
+        "description": "The operation to retrieve SAS URIs for a virtual machine's boot diagnostic logs.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine."
+          },
+          {
+            "name": "sasUriExpirationTimeInMinutes",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
```

Comment:
We should document the default value for SAS Uri expiration in case the user does not provide and also the valid range for this value,